<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kaukkis</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="MobileView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <action name="actionAddButton">
   <property name="text">
    <string>Add New Button</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
  <action name="actionPortrait">
   <property name="text">
    <string>Portrait</string>
   </property>
  </action>
  <action name="actionLandscape">
   <property name="text">
    <string>Landscape</string>
   </property>
  </action>
  <action name="actionPlugins">
   <property name="text">
    <string>Plugin Settings</string>
   </property>
  </action>
  <action name="actionAutoOrientation">
   <property name="text">
    <string>Auto Orientation</string>
   </property>
  </action>
  <action name="actionThemes">
   <property name="text">
    <string>Themes</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MobileView</class>
   <extends>QGraphicsView</extends>
   <header>mobileview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
